shell/browser/api/atom_api_web_contents.cc

```diff
@@ -894,7 +894,15 @@ void WebContents::RenderViewDeleted(content::RenderViewHost* render_view_host) {
 }
 
 void WebContents::RenderProcessGone(base::TerminationStatus status) {
-  Emit("crashed", status == base::TERMINATION_STATUS_PROCESS_WAS_KILLED);
+  // Emit the crashed event in the next tick.
+  base::PostTask(
```

Comment:
I am not sure whether to use this or `base::ThreadTaskRunnerHandle::Get()->PostTask()`

Reply:
Either should be fine, Chromium does not seem to have a preference on the style.